ui/furhat-skills/rasa/src/main/kotlin/furhatos/app/rasa/flow/interaction.kt

```diff
@@ -0,0 +1,136 @@
+package furhatos.app.rasa.flow
+
+import furhatos.flow.kotlin.State
+import furhatos.flow.kotlin.furhat
+import furhatos.flow.kotlin.onResponse
+import furhatos.flow.kotlin.state
+import furhatos.gestures.Gestures
+import io.socket.client.IO
+import io.socket.client.Socket
+import org.json.JSONObject
+import furhatos.event.Event
+
+import java.util.*
+import furhatos.event.EventSystem
+
+import java.util.regex.Pattern
+
+// CHANGE IP AND PORT HERE:
+val IP = "192.168.151.111"
+// val IP = "10.11.0.157"
```

Comment:
Replace the IP with "localhost" and remove the commented line.

Reply:
Done.